frontend/src/stores/authStore.js

```diff
@@ -6,12 +6,21 @@ const authStore = {
     username: '',
     email: '',
     roles: [],
+    clientId: '',
+    clientSecret: '',
   },
   saveState(user) {
+    console.log(user);
+
     this.state.full_name = user['full_name'];
     this.state.username = user['username'];
     this.state.email = user['email'];
     this.state.roles.push(user['scope']);
+
+    user['client_id'] ? this.state.clientId = user['client_id'] : '';
+    user['client_secret'] ? this.state.clientSecret = user['client_secret'] : '';
+
+    console.log(this.state);
```

Comment:
Do you want this console.log to be in here? Just double checking

Reply:
This has been removed as well.